runtime/README.md

```diff
@@ -312,6 +314,64 @@ Table: `INVOKE_DPE_COMMAND` output arguments
 | data_size   | u32           | Length in bytes of the valid data in the data field
 | data        | u8[...]       | DPE response structure as defined in the DPE iRoT profile.
 
+### QUOTE\_PCRS
+
+Generate a signed quote over all Caliptra hardware PCRs using the Caliptra PCR quoting key.
+All PCR values are hashed together with the nonce to produce the quote.
+
+Command Code: `0x5043_5251` ("PCRQ")
+
+Table: `QUOTE_PCRS` input arguments
+
+| **Name**     | **Type**      | **Description**
+| --------     | --------      | ---------------
+| chksum       | u32           | Checksum over other input arguments, computed by the caller. Little endian.
+| nonce        | u8[32]        | Caller-supplied nonce to be included in signed data
+
+Table: `QUOTE_PCRS` output arguments
+
+PcrValue is defined as u8[48]
+
+| **Name**     | **Type**     | **Description**
+| --------     | --------     | ---------------
+| chksum       | u32          | Checksum over other output arguments, computed by Caliptra. Little endian.
+| PCRs         | PcrValue[32] | Values of all PCRs
+| reset\_ctrs  | u32[32]      | Reset counters for all PCRs
+| signature\_r | u8[48]       | R portion of the signature over the PCR quote.
+| signature\_s | u8[48]       | S portion of the signature over the PCR quote.
+
+### EXTEND\_PCR
```

Comment:
should we note in STASH_MEASUREMENT that it also extends PCRs?

Reply:
Done